cms-publish-rest/src/main/java/se/simonsoft/cms/publish/rest/PublishStartService.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (C) 2009-2017 Simonsoft Nordic AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package se.simonsoft.cms.publish.rest;
+
+import java.util.*;
+
+import javax.inject.Inject;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import se.simonsoft.cms.item.CmsItem;
+import se.simonsoft.cms.item.CmsItemId;
+import se.simonsoft.cms.item.command.CommandRuntimeException;
+import se.simonsoft.cms.publish.config.PublishConfiguration;
+import se.simonsoft.cms.publish.config.PublishExecutor;
+import se.simonsoft.cms.publish.config.command.PublishWebhookCommandHandler;
+import se.simonsoft.cms.publish.config.databinds.config.PublishConfig;
+import se.simonsoft.cms.publish.config.databinds.job.*;
+import se.simonsoft.cms.publish.config.databinds.profiling.PublishProfilingRecipe;
+import se.simonsoft.cms.publish.config.databinds.profiling.PublishProfilingSet;
+import se.simonsoft.cms.publish.config.item.CmsItemPublish;
+import se.simonsoft.cms.release.ProfilingSet;
```

Comment:
I think we should use PublishProfilingSet.

Reply:
Right, we need the static method. Then it is correct.

These APIs need more cleanup but not this year.